packages/app/public/locales/de/common.json

```diff
@@ -25,7 +25,8 @@
   "actions": {
     "refresh": "Daten aktualisieren",
     "delete": "Löschen",
-    "cancel": "Abbrechen"
+    "cancel": "Abbrechen",
+    "backToHome": "zu Home zurück"
```

Comment:
Wouldn't be "Zurück zur Startseite" better?

Reply:
Sicherlich! (was my german alright?)

Reply:
Superlecker!